master/services/accountservice/account-service_test.go

```diff
@@ -144,3 +175,248 @@ func TestAccountService_GetByUserId(t *testing.T) {
 		})
 	}
 }
+
+func Test_accountService_Create(t *testing.T) {
+	ctx := utils.NewContextWithTraceID()
+	ctx = context.WithValue(ctx, "userID", uint(1))
+	ctx = context.WithValue(ctx, "role", uint(1))
+	type fields struct {
+		DB *gorm.DB
+	}
+	type args struct {
+		ctx     context.Context
+		account *models.Account
+	}
+
+	// 初始化数据库并进行迁移
+	db := sqlite.GetTestDB()
+	db.AutoMigrate(&models.Account{})
+
+	setup := func(db *gorm.DB) {
+		// 插入测试账户
+		testAccount := &models.Account{
+			Username: "TestUser",
+			Password: "testpassword",
+			Email:    "123@qq.com",
+			Server:   "testserver",
+			UserID:   1,
+		}
+		db.Create(testAccount)
+	}
+
+	tests := []struct {
+		name    string
+		fields  fields
+		args    args
+		wantErr bool
+	}{
+		{
+			name: "Test Create Account",
+			fields: fields{
+				DB: db,
+			},
+			args: args{
+				ctx: ctx,
+				account: &models.Account{
+					Username: "Create Account",
+					Password: "testpassword",
+					Email:    "111@qq.com",
+					Server:   "test1",
+					UserID:   1,
+				},
+			},
+			wantErr: false,
+		},
+		{
+			name: "Test Create Account with Duplicate Username",
+			fields: fields{
+				DB: db,
+			},
+			args: args{
+				ctx: ctx,
+				account: &models.Account{
+					Username: "TestUser",
+					Password: "testpassword",
+					Email:    "111@qq.com",
```

Comment:
Use better email like duplicate@qq.com or duplicate@test.com
We don't know what 111 stands for.
In the above case you can use new@test.com as well